specification/healthcareapis/resource-manager/Microsoft.HealthcareApis/preview/2018-08-20-preview/healthcare-apis.json

```diff
@@ -0,0 +1,882 @@
+      "type": "object",
+      "properties": {
+        "authority": {
+          "description": "Url to the authentication authority.",
+          "type": "string"
+        },
+        "allowedOids": {
+          "description": "List of object IDs that are allowed to access the FHIR service.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      },
+      "required": [
+        "authority",
+        "allowedOids"
+      ]
+    },
+    "ServiceCosmosDbSettingsInfo": {
+      "description": "The settings for the Cosmos DB database backing the service.",
+      "type": "object",
+      "properties": {
+        "offerThroughput": {
+          "type": "integer",
+          "description": "The provisioned throughput for the backing database.",
+          "readOnly": false
+        }
+      }
+    },
+    "ServicesSkuInfo": {
+      "description": "Information about the SKU of the service instance.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "The name of the SKU.",
+          "enum": [
+            "Standard"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "ServicesSku",
+            "modelAsString": true
+          }
+        }
+      },
+      "required": [
+        "name"
+      ]
+    },
+    "ServicesDescription": {
+      "description": "The description of the service.",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": false,
+          "description": "The common properties of a service.",
+          "$ref": "#/definitions/ServicesProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "required": [
+        "sku"
+      ]
+    },
+    "ServicesPatchDescription": {
+      "description": "The description of the service.",
+      "type": "object",
+      "properties": {
+        "tags": {
+          "description": "Instance tags",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "allOf": {
+          "description": "Sku details of a service.",
+          "$ref": "#/definitions/ServicesSkuInfo"
+        }
+      }
+    },
+    "Resource": {
+      "description": "The common properties of a service.",
+      "properties": {
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The resource identifier."
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The resource name.",
+          "pattern": "^[a-zA-Z0-9][a-zA-Z0-9-]{1,21}[a-zA-Z0-9]$"
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The resource type."
+        },
+        "kind": {
```

Comment:
Are you using kind or etag? Any optional properties you aren't using should be removed.